remme/tp/account.py

```diff
@@ -97,7 +108,11 @@ def _transfer(self, context, public_key, transfer_payload):
         References:
             - https://github.com/Remmeauth/remme-core/blob/dev/remme/genesis/__main__.py
         """
-        address = self.make_address_from_data(public_key)
+        if transfer_payload.sender_account_type == TransferPayload.SenderAccountType.Value('ACCOUNT'):
+            address = self.make_address_from_data(public_key)
+
+        else:  # sender account type is node account
```

Comment:
This looks looks like it allows to pass anything different from `TransferPayload.SenderAccountType.Value('ACCOUNT')`.

Reply:
@eugene-babichenko, we have a [test](https://github.com/Remmeauth/remme-core/blob/aa3e42e67e833ec9dd8f9a4632a774474ddc880d/testing/unit/tp/account/test_transfer.py#L124) for checking sender account type value.